backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/json/JsonMergePatchHttpMessageConverter.java

```diff
@@ -27,8 +29,8 @@ protected boolean supports(Class<?> clazz) {
 
 	@Override
 	protected JsonMergePatch readInternal(Class<? extends JsonMergePatch> clazz, HttpInputMessage httpInputMessage) throws IOException, HttpMessageNotReadableException {
-		try {
-			return Json.createMergePatch(Json.createReader(httpInputMessage.getBody()).readValue());
+		try (JsonReader jsonReader = Json.createReader(httpInputMessage.getBody())) {
```

Comment:
```suggestion
		try (final var jsonReader = Json.createReader(httpInputMessage.getBody())) {
```

Reply:
Implemented in commit 74e3d8d4dbedcbce2497740fdcf756f4ac2d3d92.